dynawo/sources/Modeler/DataInterface/DYNDataInterfaceIIDM.cpp

```diff
@@ -977,6 +1001,15 @@ DataInterfaceIIDM::mapConnections() {
     }
   }
 
+  const vector<shared_ptr<HvdcLineInterface> >& hvdcs = network_->getHvdcLines();
+  vector<shared_ptr<HvdcLineInterface> >::const_iterator iHvdc;
```

Comment:
Please include the iterator inside the for loop

Reply:
Can you be more specific? How do I define the for loop then?

Reply:
for (vector<shared_ptr<HvdcLineInterface> >::const_iterator iHvdc = hvdcs.begin(); iHvdc != hvdcs.end(); ++iHvdc) {

Reply:
thanks, done